kura/org.eclipse.kura.util/META-INF/MANIFEST.MF

```diff
@@ -2,17 +2,19 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: Kura Base Utilities
 Bundle-SymbolicName: org.eclipse.kura.util;singleton:=true
-Bundle-Version: 1.0.0.qualifier
+Bundle-Version: 1.0.1.qualifier
 Bundle-Vendor: Eclipse Kura
 Bundle-License: Eclipse Public License v1.0
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
-Import-Package: org.eclipse.kura;version="[1.2,2.0)",
+Import-Package: org.apache.commons.io;version="[2.4,3.0)",
+ org.eclipse.kura;version="[1.2,2.0)",
  org.eclipse.kura.annotation;version="[1.0,2.0)",
  org.eclipse.kura.localization;version="[1.0,2.0)",
  org.eclipse.kura.localization.resources;version="[1.0,2.0)",
  org.osgi.framework;version="[1.7.0,2.0.0)",
  org.slf4j;version="1.6.4"
-Export-Package: org.eclipse.kura.util.base;version="1.0.0",
+Export-Package: org.eclipse.kura.core.util; version="1.1.0", 
+ org.eclipse.kura.util.base;version="1.0.0",
```

Comment:
In the recently introduced Utility bundle, the utilities are packaged in a cohesive way that is reflected by the assigned name, such as `util.base`, `util.collection`, `util.service` etc. According to my opinion, you can create new packages as follows:

- `util.io` -> `GZipUtil`, `IOUtil`
- `util.net` -> `NetUtil`
- `util.process` -> `ProcessUtil`, `SafeProcess`
- `util.base` (already existent) -> `NamedThreadFactory`, `QuickSort`, `ThrowableUtil`, `ValidationUtil`